types/artifacts.d.ts

```diff
@@ -179,6 +181,21 @@ declare global {
         params: {name: string; value: string}[];
       }
 
+      export interface IFrameElement {
+        /** The `id` attribute of the IFrame. */
+        id: string,
+        /** The `src` attribute of the IFrame. */
+        src: string,
+        /** The IFrame's ClientRect. @see https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect */
+        clientRect: ClientRect,
+        /** The visibility of the IFrame. */
+        isVisible: boolean,
+        /** If the IFrame or an ancestor of the IFrame is fixed in position. */
+        isPositionFixed: boolean,
+        /** The Frame of the IFrame. @see https://chromedevtools.github.io/devtools-protocol/tot/Page#type-Frame */
+        frame: Crdp.Page.Frame,
```

Comment:
what's this used for? It seems like a lot of it isn't going to be usable after the browser connection is closed.

Reply:
aye. i'm not seeing this used in the audits right now.

Reply:
It's used in the paint timing audit, to match the paint trace events with corresponding iframes